docs/UserGuide.md

```diff
@@ -201,11 +219,11 @@ _Details coming soon ..._
 |------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | **Add**    | `add n/RECIPE_NAME [d/RECIPE_DURATION] [p/RECIPE_PORTION] [t/RECIPE_TAGS]...  [i/-n INGREDIENT_NAME [-a INGREDIENT_AMOUNT] [-e {ESTIMATED AMOUNT}] [-cn {COMMON NAME}] [-r {REMARKS}]... [-s {SUBSTITUTION}]...]... [s/RECIPE_STEPS]...` <br/> e.g., `add n/Chicken Noodles d/20 minutes p/1-2 people i/-n chicken thigh -a 300g i/-n noodles i/-n soy sauce -a 2 tablespoons -s salt` |                                                                                                                                                             |            |                                      |
 | **List**   | `list`                                                                                                                                                                                                                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                                                             |            |                                      |
-| **Find**   | `find KEYWORD [KEYWORDS]...    `<br/> e.g., `find cheese rice`                                                                                                                                                                                                                                                                                                                         |
+| **Find**   | `find [PREDICATE] KEYWORD [KEYWORDS]...    ` for predicate: `name`, `tag` <br/> e.g., `find cheese rice`, `find name popcorn`, `find tag western`                                                                                                                                                                                                                                      |
```

Comment:
rename PREDICATE to PROPERTY and KEYWORD to ADDITIONAL KEYWORDS to keep it consistent with the syntax we gave in the above section on the "find" command